endoreg_db/models/requirement/requirement_evaluation/__init__.py

```
@@ -0,0 +1,75 @@
+    FindingIntervention,
+    FindingLocationClassification,
+    Requirement,
+)
+
+
+def get_kwargs_by_req_type_and_operator(requirement, **kwargs):
+    pass
+
+
+def models_match_all(requirement: Requirement, **kwargs):
+    models_dict = requirement.get_models_dict()
```

Ruff (0.8.2)

Local variable `models_dict` is assigned to but never used

Remove assignment to unused variable `models_dict`

(F841)